scripts/hic_table.py

```diff
@@ -45,6 +45,10 @@ def get_args():
                         default=Path("~/keypairs.json").expanduser(),
                         help="The keypair file. Default is --keyfile={}".format(
                             Path("~/keypairs.json").expanduser()))
+    parser.add_argument('--dryrun',
+                        default=False,
```

Comment:
You should add "action='store_true'," when setting up this argument - that way you just need to indicate --dryrun with no value and it will set the value to True.